entry_point.py

```diff
@@ -59,6 +61,66 @@ def change_values(file_name, getter_func):
         print(new_str.replace('\n', ''))
 
 
+def get_varname(var, prefix="ODOORC_"):
+    """
+    Cleans the string and return the variable ready to use in the config file
+
+    :param var: variable name as comes from the source
+    :param prefix: prefix to be deleted from the string
+    :return: Sting in lowercase without the prefix
+    """
+    r = re.compile("^"+prefix)
+    return r.sub("", var, 1).lower()
+
+
+def get_odoo_vars(getter_func):
+    res = dict()
+    for key, value in getter_func():
```

Comment:
@ruiztulio Which getters can be used here?, Only the `environ.items` pointer is passed to this metod, and only the `append_values` method calls this method, we can avoid passing this parameter to this method and to the `append_values` method if we use `for key, value in environ.items():` directly. Or are there other methods outside this file that need to pass a different getter?

Reply:
@Angelfentanez We have plans to get the configuration from external sources too, so we need to pass the function that will get the parameters (initially was redis, but it was removed)

Reply:
@Angelfentanez after I wrote it I realized that we don't know if the source will give us the variables in upper or lowecase, so it was changed to work everything a lower from the very beginning